### ISSUE TITLE
Allow "ambiguous" splats for core/stdlib methods

### ISSUE THREAD

**Issue by emschwar** [bbatsov/rubocop#2000]
I have a fair amount of code that looks like

```ruby
class Foo
  PROPERTIES = [
    :a,
    :b
  ]
  attr_reader *PROPERTIES
end
```

This is getting caught by the AmbiguousOperator cop, but I think the clause from the style guide that says,
>Omit parentheses around parameters for methods that are part of an internal DSL (e.g. Rake, Rails, RSpec), methods that have "keyword" status in Ruby (e.g. attr_reader, puts) and attribute access methods.

should govern here. Certainly in the case of `attr_reader` (and its various cousins, `attr_accessor`, et. al), it's clear that the splat is not ambiguous.

**Comment by bbatsov** [comment]
I understand your logic, but this is how MRI's built-in lint operates as well. Plus, there's no way to know which method is which using static analysis - we can only guess, which yields false positives from time to time.
